assume/common/forecasts.py

```diff
@@ -313,16 +315,19 @@ def calculate_marginal_cost(self, pp_series: pd.Series):
         if fp_column in self.forecasts.columns:
             fuel_price = self.forecasts[fp_column]
         else:
-            fuel_price = pd.Series(0, index=self.index)
+            fuel_price = pd.Series(0.0, index=self.index)
 
         emission_factor = pp_series["emission_factor"]
         co2_price = self.forecasts["fuel_price_co2"]
 
         fuel_cost = fuel_price / pp_series["efficiency"]
         emissions_cost = co2_price * emission_factor / pp_series["efficiency"]
         fixed_cost = pp_series["fixed_cost"] if "fixed_cost" in pp_series else 0.0
+        variable_cost = (
+            pp_series["variable_cost"] if "variable_cost" in pp_series else 0.0
+        )
 
-        marginal_cost = fuel_cost + emissions_cost + fixed_cost
+        marginal_cost = fuel_cost + emissions_cost + fixed_cost + variable_cost
```

Comment:
The confusion for me is rather, that we use the fixed costs in the marginal costs. Usually, fixed costs do not depend on the output, right?
So therefore I used variable costs to include costs for renewable energies, which do not have any fuel costs ect.
I think a dependency on the time is not a problem, since we have the same for fuel costs: they can be defined both as float and time series.
So I would suggest using both fixed and variable costs, but removing fixed costs from the marginal costs. They could then still be used in some strategies.
And costs relative to the price would rather be something like an interest rate, or not?

Reply:
Clarify usage of fixed_cost, variable_cost and relative_margin

As marginal cost is the change in the total cost that arises when the quantity produced is incremented by one unit - it should not include fixed_costs which must be payed every hour.

Variable cost as suggested here covers the case of having an additional cost on top of the price calculation, which is respected in the marginal cost - this could also be seen as runtime cost in €/MW of the unit

Additionally, it would be good to have a relative margin on the price, which should be added after adding the fixed_cost/variable_cost, by multiplying the cost.

We should document that and make sure that this works as expected :)